.github/workflows/gcp-filings-noebook-reort-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: GCP Filings Notebook Report CI
+#Note job will fail tests as a default db is not setup
+on:
+  push:
```

Comment:
Ah was wondering why this wasn't running. This should be:
```
pull_request:
    types: [assigned, synchronize]
    paths:
      - "gcp-jobs/filings-notebook-report/**"
```

Reply:
chaged